app/src/main/assets/AnimalOverlaysConfig.json

```diff
@@ -0,0 +1,10 @@
+{
+	"animalOverlays" : {
```

Comment:
I had it as an array but somehow the automatic mapping didn't work then. http://stackoverflow.com/questions/16377754/parse-json-file-using-gson but if you know how to do it, I will change it. I just don't know :)